semgrep-core/src/parsing/pfff/json_to_generic.ml

```diff
@@ -102,5 +102,10 @@ let program ?unescape_strings ast =
 let any x =
   match x with
   | E e -> G.E (expr e)
-  | PartialSingleField (v1, v2, v3) ->
-      G.Partial (G.PartialSingleField (v1, v2, expr v3))
+  | PartialSingleField (v1, _v2, v3) ->
```

Comment:
Add a comment that this code copies how the expr function translates fields of a dictionary, and that in particular metavariables need to be translated properly to allow unification